<?xml version="1.0" ?>
<sdf version="1.5">
  <model name="mpnet_coffee">
    <pose frame="world">1 0 0.0 1.57079632679 0 0</pose>
    <static>true</static>
    <link name="link">
      <inertial>
        <mass>0.1</mass>
      </inertial>
      <visual name="visual">
        <geometry>
          <mesh>
            <uri>model://mpnet_coffee/meshes/coffee.stl</uri>
            <scale>0.035 0.035 0.035</scale>
          </mesh>
        </geometry>
      </visual>
    </link>
  </model>
</sdf>
